code/obj/item/uplinks.dm

```diff
@@ -779,6 +779,16 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 	spy
 		purchase_flags = UPLINK_SPY
 
+///Datum used to combine the bounty being claimed with the item being delivered
+/datum/bounty_claim
+	var/datum/bounty_item/bounty = null
+	var/atom/delivery = null
+
+	New(datum/bounty_item/bounty, atom/delivery)
```

Comment:
disposing

Reply:
Hope that's all I need to do? I don't want to dispose the properties as this is just a datum for combining pre-existing arguments into one return.